packages/neuron-ui/src/components/PricePanel/index.tsx

```diff
@@ -107,11 +115,35 @@ const PricePanel: React.FunctionComponent<PricePanelProps> = ({ price, field, on
     [priceArray, feeValuesArray]
   )
 
+  const batchGenerateExperimental = useCallback(async () => {
+    if (experimental && experimental?.params) {
+      const generator = getGenerator(experimental.params?.sendType)
+      try {
+        const requestArray = priceArray.map(itemPrice => generator({ ...experimental.params, feeRate: itemPrice }))
+        const allPromiseResult = await Promise.allSettled(requestArray)
+
+        const resList = allPromiseResult.map((batchItem, index: number) => ({
+          feeRateValue: priceArray[index],
+          // @ts-ignore
+          feeValue: batchItem.value.result?.fee,
```

Comment:
```typescript
interface PromiseFulfilledResult<T> {
    status: "fulfilled";
    value: T;
}

interface PromiseRejectedResult {
    status: "rejected";
    reason: any;
}

type PromiseSettledResult<T> = PromiseFulfilledResult<T> | PromiseRejectedResult;
```

The error here is because there is indeed no value in `PromiseRejectedResult`, and `ts-ignore` should not be used to break the type system.
If you think it is impossible to have an error, then you should use `Promise.all` instead of `Promise.allSettled`.
If you think there may be errors, but you want to ignore them, then create an `isFulfilled` utility function and do a `.filter(isFulfilled)` before `.map`.

```typescript
const isFulfilled = <T extends any>(input: PromiseSettledResult<T>): input is PromiseFulfilledResult<T> =>
          input.status === 'fulfilled'
```